roles/rhn-subscription/defaults/main.yml

```diff
@@ -0,0 +1,9 @@
+---
+rhn_subscription:
+  username: ~
+  password: ~
+  pool_names_regex: ^$
```

Comment:
can we have this default to  "^(Red Hat OpenStack Platform)" and let the osds and swifts override ?
I'd vote for the same behavior w/ the repos -- that the default set can be in the defaults so compute and controller group_vars don't need to override.